validation.rb

```diff
@@ -0,0 +1,79 @@
+require 'date'
+
+module Validation
+  def self.match(pattern, text)
```

Comment:
A module is a collection of methods, variables, and constants stored in a container. In most cases, we don't use class-level methods in classes. 
Therefore, please remove the self. prefix before the method name.